src/Avalonia.Base/Input/Platform/KeyGestureFormatInfo.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Avalonia.Input.Platform
+{
+
+    /// <summary>
+    /// Provides platform specific formatting information for the KeyGesture class
+    /// </summary>
+    /// <param name="platformKeyOverrides">A dictionary of Key to String overrides for specific characters, for example Key.Left to "Left Arrow" or "←" on Mac.
+    /// A null value is assumed to be the Invariant, so the included set of common overrides will be skipped if this is null.  If only the common overrides are
+    /// desired, pass an empty Dictionary instead.</param>
+    /// <param name="meta">The string to use for the Meta modifier, defaults to "Cmd"</param>
+    /// <param name="ctrl">The string to use for the Ctrl modifier, defaults to "Ctrl"</param>
+    /// <param name="alt">The string to use for the Alt modifier, defaults to "Alt"</param>
+    /// <param name="shift">The string to use for the Shift modifier, defaults to "Shift"</param>
+    public sealed class KeyGestureFormatInfo(Dictionary<Key, string>? platformKeyOverrides = null,
```

Comment:
Since it's a public API, the first parameter should probably be a `IReadOnlyDictionary` instead of a `Dictionary`.

Reply:
You're right.  I've done that, plus moving the non-Apple overrides to the common dictionary as discussed above.  Also at some point I accidentally committed an addition I'd made to the MenuPage in the ControlCatalog, so I'll undo that as well.